docs/howto/custom-file-storage.txt

```diff
@@ -88,8 +88,9 @@ instead).
 .. method:: get_valid_name(name)
 
 Returns a filename suitable for use with the underlying storage system. The
-``name`` argument passed to this method is the original filename sent to the
-server, after having any path information removed. Override this to customize
+``name`` argument passed to this method is either the original filename sent to 
+the server or the filename returned by ``upload_to`` if ``upload_to`` was a 
+callable, after having any path information removed. Override this to customize
```

Comment:
Should have a ``.. versionchanged:: 1.9`` annotation below this paragraph with a description of the change, as well as a mention in the 1.9 release notes.